src/spikeinterface/core/node_pipeline.py

```diff
@@ -627,7 +663,7 @@ def _compute_peak_pipeline_chunk(segment_index, start_frame, end_frame, worker_c
         )
         # compute the graph
         pipeline_outputs = {}
-        for node in nodes:
+        for i, node in enumerate(nodes):
```

Comment:
```suggestion
        for node in nodes:
```
Tiny thing: you end up not using `i`.

Reply:
Thanks!